### PR TITLE
Add pre-commit!

### PR DESCRIPTION
**Pull request by AetherUnbound**
Lots of whitespace changes, so be sure to check "Hide whitespace changes" when reviewing

### CODE REVIEW

**Comment by AetherUnbound** [comment]
Ah crap I gotta update the GitHub PR workflow. Something for tomorrow.

File: signal_scanner_bot/env.py [hunk 1]
```diff
@@ -31,7 +30,8 @@ def _env(
     fail: bool = True,
     default: Any = None,
 ) -> Any:
-    """
+    """Get environment variable.
```
Comment (sarayourfriend) [inline signal_scanner_bot/env.py:33]: Are these really meant to be on the same line as the opening `"""`?

**Review by sarayourfriend — APPROVED** [review]
LGTM! 🚀

File: signal_scanner_bot/env.py (hunk 1 above)
Comment (AetherUnbound) [reply signal_scanner_bot/env.py:33]: That's the way it's "supposed" to be (and how we do for work) but honestly, I hate it! 😄 I'm gonna change this rule lol

**Comment by AetherUnbound** [comment]
Okay this should be ready again! Sorry it took so long :) Next up, justfile!

**Review by sarayourfriend — APPROVED** [review]

**Pull request merged**